linux/user-and-file-management/manipulating-environment-variables/manipulating-environment-variables.md

```diff
@@ -15,59 +9,68 @@ type: normal
 
 category: must-know
 
-inAlgoPool: false
-
 tags:
-
   - introduction
+  - shell
+  - environment
+  - variables
 
 
 ---
 
-# Manipulating Environment Variables (Part 1)
+# Shell vs Environment
 
 ---
 ## Content
 
+Shell and environment variables are slightly different.
 
-### Shell Variable vs Environment Variable
-
-To understand what an *environment* variable is, we first have to understand what a *shell* variable is.
-
-You can define a new shell variable using the `=` operator, like so:
+You can define a new shell variable using the `=` operator:
 
 ```shell
-enki@host ~ $ DRINK="water"
-enki@host ~ $ echo $DRINK
+drink="water"
+echo $drink
 water
-enki@host ~ $
 ```
 
 Shell variables are used only by the shell itself, and not available to any of its children processes. They are commonly used in scripts to make calculations and to store intermediate results.
 
-```shell
-enki@host ~ $ DRINK="water"
-enki@host ~ $ AMT="plenty"
-enki@host ~ $ ADVICE="drink $AMT of $DRINK"
-enki@host ~ $ echo $ADVICE
-drink plenty of water
-enki@host ~ $
-```
-
 Environment variables are shell variables that have been exported, being available to its child process. We achieve this using the `export` command:
 
 ```shell
-enki@host ~ $ export DRINK="water"
-enki@host ~ $ echo $DRINK
+export DRINK="water"
+echo $DRINK
 water
-enki@host ~ $
 ```
 
-There are four things you might want to do to an environment variable, which we'll refer to as "variable" from here on out:
+Notice anything different? To avoid overwriting an important environment variable, it is **strongly recommended to write them in uppercase**. This way, you don't need to remember all the environment variables used by your system, you can look at the letter case.
+
+> **Remember:** if it's your variable **lowercase it**. If you export it, **uppercase it**.
```

Comment:
This is a bit confusing? 

`DRINK` is uppercase the whole time in this insight. There are no lowercase variables.

I added a prior comment to mention the uppercase naming convention in one of the first insights in this workout. 

Let's simplify this like this:

```suggestion
> ⚠️When exporting a variable into the environment, be careful not to overwrite another important environment variable. 
```